nmoscommon/webapi.py

```diff
@@ -571,112 +617,127 @@ def inner(*args, **kwargs):
                 return f(*args, **kwargs)
             return inner
 
-        def getbases(cl):
-            bases = list(cl.__bases__)
-            for x in cl.__bases__:
+        def getbases(cls):
+            bases = list(cls.__bases__)
+            for x in cls.__bases__:
                 bases += getbases(x)
             return bases
 
-        for klass in [routesObject.__class__,] + getbases(routesObject.__class__):
-            for name in klass.__dict__.keys():
-                value = getattr(routesObject, name)
-                if callable(value):
-                    endpoint = "{}_{}".format(basepath.replace('/', '_'), value.__name__)
-                    if hasattr(value, 'app_methods') and value.app_methods is not None:
-                        methods = value.app_methods
+        for cls in [routesClass.__class__, ] + getbases(routesClass.__class__):
+            for attr in cls.__dict__.keys():
+                routesFunction = getattr(routesClass, attr)
```

Comment:
Being picky, is this not a method rather than function? I realise it could also be a variable, but if it's only checked within callable then I can see why this name makes sense.

Reply:
Yeah it can be an attribute but for the sake of brevity I avoided something like:
routesAttr = getatter(routesClass, attr)
if callable(routesAttr):
   routesFunction = routesAttr

but I can add it if you want? If not then I'll just change it to routesMethod

Reply:
No I think just changing to method is fine